realm/realm-library/src/androidTest/java/io/realm/LinkingObjectsManagedTests.java

```diff
@@ -151,44 +151,6 @@ public void basic_multipleReferencesFromParentList() {
         assertEquals(parent, child.getListParents().last());
     }
 
-    // A listener registered on the backlinked object should be called when a commit adds a backlink
-    @Test
-    @RunTestInLooperThread
-    public void notification_onCommitModelObject() {
```

Comment:
Technically the `RealmResults` field on `child` is a field with a (backlink) reference to the parent, so one could argue it should trigger the same type of notification as when a `RealmList` changes. I'm fine with leaving it as it is right now, but we should probably document that somewhere.